cmd/tailscale/main.go

```diff
@@ -675,10 +679,19 @@ func (s *BackendState) updateExitNodes() {
 		myExit := p.StableID() == exitID
 		hasMyExit = hasMyExit || myExit
 		exit := Peer{
-			Label:  p.DisplayName(true),
-			Online: canRoute,
-			ID:     p.StableID(),
+			Label:    p.DisplayName(true),
+			Online:   canRoute,
+			ID:       p.StableID(),
+			Location: p.Hostinfo().Location(),
```

Comment:
Right now we call updateExitNodes under three circumstances: 
-the prefs change
-the netmap changes
-the user wants to use an exit node
Is there any case where the location for an exit node might change, but none of the three cases above is triggered? Or is it ok for the location to be stale because we never really use it anyway except for filtering/sorting exit nodes for display?

Reply:
Nope - The location is sent from control, so the netmap will have to be resent in that case.